File: client/src/dto/importRouteDTO.js
export class ImportRoute {
  constructor({
    id = null,
    name = "",
    waypoints = "",
    geoCoordinates = "",
    createdTime = null,
    routeId = null,
    personId = null,
  } = {}) {
    this.id = id;
    this.name = name;
    this.waypoints = waypoints;
    this.geoCoordinates = geoCoordinates;
    this.createdTime = createdTime;
    this.routeId = routeId;
    this.personId = personId;
  }
}

File: routes/src/main/java/com/peak/routes/dto/ImportRouteDTO.java
package com.peak.routes.dto;

import java.time.LocalDateTime;

public class ImportRouteDTO extends BaseRouteDTO {
    private Long routeId;

    public ImportRouteDTO() {
    }

    public ImportRouteDTO(Long id, String name, String waypoints, String geoCoordinates, LocalDateTime createdTime, Long personId, Long routeId) {
        super(id, name, waypoints, geoCoordinates, createdTime, personId);
        this.routeId = routeId;
    }

    public Long getRouteId() {
        return routeId;
    }

    public void setRouteId(Long routeId) {
        this.routeId = routeId;
    }
}
